### DISCUSSION TITLE
[Feature] Ability for admin to allocate storage space for each user

### DISCUSSION THREAD

**Discussion by junglebells** [immich-app/immich#5309] [Feature Request]
### The feature

I'd love the possibility of adding some of my extended family members on the platform. However, would like to limit how much disk space is available to them, and also display how much space they have actually used.

Currently, the storage stats (on bottom left) are misleading. It displays total space used and available on the server, not by user. Ideally, it should be the quota allocated to each user.

### Platform

- [ ] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #1642
